src/test/java/weblogic/logging/exporter/FirstTest.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+ */
+
+package weblogic.logging.exporter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("The first test")
```

Comment:
Do you really mean to check this in? It looks as though you were experimenting with JUnit.

Reply:
No I didn't